tests/nimdoc/t17615.nim

```diff
@@ -0,0 +1,12 @@
+discard """
+  cmd: "nim doc -r $file"
+  errormsg: "Runnable Examples cannot stay behind the actual code"
+  line: 10
+"""
+
+func big*(integer: int): int =
```

Comment:
minimized bugs in regression tests are better (my bad, i should've minimized https://github.com/nim-lang/Nim/issues/17615 when i reported the issue)